resources/views/auth/register.blade.php

```diff
@@ -1,52 +1,87 @@
 <x-guest-layout>
-    <form method="POST" action="{{ route('register') }}">
-        @csrf
 
-        <!-- Name -->
-        <div>
-            <x-input-label for="name" :value="__('Name')" />
-            <x-text-input id="name" class="block mt-1 w-full" type="text" name="name" :value="old('name')" required autofocus autocomplete="name" />
-            <x-input-error :messages="$errors->get('name')" class="mt-2" />
+<div id="appCapsule" class="bg-[#0d4656] flex justify-center items-center h-full min-h-screen">
+        <div class="login-form mt-1 w-full">
+            <div class="flex flex-col gap-y-6 text-center mb-8">
+                {{-- <img src="{{ asset('assets/img/logo-slim.png') }}"
+                    style="position:relative;width:100%;margin-top:15px;" /> --}}
+                <h1 class="text-white text-4xl font-bold">Register</h1>
+                <h4 class="text-gray-100">
+                    Create your free account!
+                    Already have an account? <a href="{{ route('login') }}" class="text-blue-500">Login</a>
+                </h4>
+            </div>
+ 
+            <div class="section mb-3 mt-2 w-full lg:w-1/2 mx-auto">
+                <form action="{{ route('register') }}" class="" method="POST">
+                    @csrf
+                    <div class="flex justify-between flex-wrap gap-y-2 p-8">
+                        <div class="w-full lg:w-[45%] flex flex-col gap-y-2">
+                            <label class="form-label text-gray-200 fs-6" for="first_name">First name <span
+                                    class="text-danger"></span></label>
+                            <input class="rounded-xl" id="first_name" name="first_name" required type="text">
+                        </div>
+ 
+                        <div class="w-full lg:w-[45%] flex flex-col gap-y-2">
+                            <label class="form-label text-gray-200 fs-6" for="last_name">Last name <span
+                                    class="text-danger"></span></label>
+                            <input class="rounded-xl" id="last_name" name="last_name" required type="text">
+                        </div>
+ 
+                        <div class="w-full lg:w-[45%] flex flex-col gap-y-2">
+                            <label class="form-label text-gray-200 fs-6" for="phone">Phone number <span
+                                    class="text-danger"></span></label>
+                            <input class="rounded-xl" id="phone"required name="phone" type="tel" required>
```

Comment:
```suggestion
                            <input class="rounded-xl" id="phone"required name="phone" type="tel" required>
```
Please check and remove the duplicate 'required' attribute